src/utils/hoc.js

```diff
@@ -37,3 +38,6 @@ export let contexturify = _.flow(
 export let withTreeLens = Component => ({ prop = 'value', ...props }) => (
   <Component {...{ lens: props.tree.lens(props.node.path, prop), ...props }} />
 )
+
+export let getDisplayName = Component =>
+  F.cascade(['displayName', 'name'], Component) || 'Unknown'
```

Comment:
also, since I think it will default to unknown if undefined, you can probably just make this

```js
export let getDisplayName = F.cascade(['displayName', 'name'])
```

Reply:
It displays `undefined`, so I'm leaving it as `Unknown` to feel more react-dev tools-like.